app/models/validators/utaac_decision_validator.rb

```diff
@@ -12,6 +13,6 @@ class UtaacDecisionValidator < SimpleDelegator
   validates :tribunal_decision_category, presence: true
   validates :tribunal_decision_decision_date, presence: true, date: true
   validates :tribunal_decision_judges, presence: true
-  validates :tribunal_decision_sub_category, presence: true
+  validates :tribunal_decision_sub_category, presence: true, tribunal_decision_sub_category: true
```

Comment:
Rails uses this symbol to determine the name of the validator class. e.g. Rails uses the `DateValidator` class when `date: true`, the `TribunalDecisionSubCategoryValidator` class when `tribunal_decision_sub_category: true`, etc.

Based on our meeting this morning I'm working on extending the validator to perform additional checks beyond the parent category check. Think `TribunalDecisionSubCategoryValidator` / `tribunal_decision_sub_category: true` still works, although awkward. What do you think?